docs/contributor/04-30-ci-cd.md

```diff
@@ -42,6 +42,7 @@ The following CI jobs are part of the development cycle.
 |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | [`Pull Request / Wait for image build`](https://github.com/kyma-project/api-gateway/blob/dda6608241107d653e68fec8503ce9489eab57f3/.github/workflows/pull-request.yaml#L40)                           | Waits for the operator image build to finish so that the image-dependent jobs can be executed.                                                               |
```

Comment:
As far as I understand, this was replaced with the `build-image` job.

Reply:
Yes, reverted too much, wait job is replaced by the build one, `pull-api-gateway-manager-build` still in there.